lua/rocks/operations.lua

```diff
@@ -309,6 +321,10 @@ operations.add = function(rock_name, version)
                 user_rocks.plugins = {}
             end
 
+            -- Set installed version as `scm-1` if development version has been installed
+            if version == "dev" then
+                installed_rock.version = "scm-1"
+            end
```

Comment:
question: What if the user called `:Rocks install <plugin> dev`? Does that result in the version being saved as `scm-1`?
If so, I think just saving it as `"dev"` is fine.

Reply:
> What if the user called `:Rocks install <plugin> dev`? Does that result in the version being saved as `scm-1`?

Yup, the plugin is stored as `<plugin> = "scm-1"` in the `rocks.toml` file. If the version is saved as `dev` instead it would cause errors when trying to update or sync since the regex we use to check the versions will not match anything